pipelines/ligtheweight-component-and-container-operations/ligtheweight-component-and-container-operations.ipynb

```diff
@@ -0,0 +1,233 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Combining pythong lightweigt components and container operations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demos: \n",
+    "\n",
+    "* Defining a Kubeflow pipeline with the KFP SDK, combinging python lightweight components operations and container operations\n",
+    "* Creating an experiment and submitting pipelines to the KFP run time environment using the KFP SDK \n",
+    "\n",
+    "Reference documentation: \n",
+    "* https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/\n",
+    "* https://www.kubeflow.org/docs/pipelines/sdk/build-component/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Prerequisites: Install or update the pipelines SDK\n",
+    "\n",
+    "You may need to **restart your notebook kernel** after updating the KFP sdk.\n",
+    "\n",
+    "This notebook is intended to be run from a Kubeflow notebook server. (From other environments, you would need to pass different arguments to the `kfp.Client` constructor.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# You may need to restart your notebook kernel after updating \n",
+    "!python3 -m pip install kfp-server-api --upgrade --user\n",
+    "!python3 -m pip install kfp --upgrade --user"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "EXPERIMENT_NAME = 'Combining pythong lightweigt components and container operations'        # Name of the experiment in the UI\n",
+    "BASE_IMAGE = 'tensorflow/tensorflow:2.0.0b0-py3'    # Base image used for components in the pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import kfp\n",
+    "import kfp.dsl as dsl\n",
+    "from kfp import compiler\n",
+    "from kfp import components\n",
+    "from kfp import gcp\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create pipeline component"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Create a python function"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@dsl.python_component(\n",
```

Comment:
This decorator is being deprecated.
`func_to_container_op` will just use the function docstring.